src/parser/priest/discipline/modules/features/AtonementDamageSource.ts

```diff
@@ -10,10 +10,13 @@ class AtonementDamageSource extends Analyzer {
     return this._event;
   }
 
-  on_damage(event: DamageEvent) {
-    if (!this.owner.byPlayer(event) && !this.owner.byPlayerPet(event)) {
-      return;
-    }
+  constructor(options: Options){
+    super(options);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER), this.onDamage);
```

Comment:
```suggestion
    this.addEventListener(Events.damage.by(SELECTED_PLAYER | SELECTED_PLAYER_PET), this.onDamage);
```

I just remembered these were masks, so probably missed it in some other files. just saves a line or two